kotekan/kotekan.cpp

```diff
@@ -178,13 +189,13 @@ bool set_gps_time(Config &config) {
  * @brief Starts a new kotekan mode (config instance)
  *
  * @param config The config to generate the instance from
- * @param requires_gps_time If set to true, then the config must provide a valid time
- *                          otherwise an error is thrown.
+ * @param requires_gps_time If set to true, then the config must provide a valid
```

Comment:
From a readably point of view I prefer having the second line indented to the start of the first line.   Although I'm guessing this might be hard to enforce with `clang-format`.

Reply:
We could tell clang-format not to touch `@param`. That would only result in some still exceeding the 100 columns.

Reply:
In the end this didn't work, because it would still butcher the line after the pragma. We can still indent by hand after clang-format took care of overlength lines. It should leave it like that afterwards.